packages/hash/api/src/db/postgres/aggregation/updateAggregationOperation.ts

```diff
@@ -54,6 +43,15 @@ export const updateAggregationOperation = (
       return dbEntity;
     });
 
+    const { operation } = params;
+
+    const updatedDbAggregation = {
+      ...previousDbAggregation,
+      operation,
+      createdByAccountId,
```

Comment:
Should the `createdByAccountId` change when updating? I see why we would assign `createdByAccountId` and `createdAt` for the aggregation in `updatedDbAggregation` if it is in `previousDbAggregation`

Reply:
No we should not, but I planned to address this as part of [this follow up task](https://app.asana.com/0/1201959586244685/1202032424659148/f) as it introduces the relevant `created_at/by` `updated_at/by` columns we want in the `aggregations` and `aggregation_versions` tables.

Reply:
Perfect, that sounds great!